models/token.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    token: Joi
+        .string()
+        .token(),
+
+    id: Joi
+        .number()
+        .integer()
+        .positive(),
+
+    userId: Joi
+        .number()
+        .integer()
+        .positive()
+        .description('User ID'),
+
+    name: Joi
```

Comment:
Token names should probably be very similar to secret names.

https://github.com/screwdriver-cd/data-schema/blob/master/config/job.js#L23